### PR TITLE
Add response statuses to Vote and Comment models

### PR DESCRIPTION
**Pull request by JjuliaP**
Add response statuses to Vote Model
Add response statuses to Comment Model

## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVPYTP-92)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/vote/views.py [hunk 1]
```diff
@@ -286,19 +290,19 @@ def put(self, request, team_id, event_id, vote_id, answer_id=None):  # pylint: d
...
 
         if not answer_id:
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
 
         team_members = [user.id for user in team.members.all()]
 
         if user.id not in team_members:
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
```
Comment (redmyt) [inline eventually/vote/views.py]: maybe it is better to use this `access denied` response?

File: eventually/vote/views.py [hunk 2]
```diff
@@ -337,15 +341,15 @@ def delete(self, request, team_id, event_id, vote_id, answer_id=None):  # pylint
         team = Team.get_by_id(team_id)
 
         if not answer_id:
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
 
         team_members = [user.id for user in team.members.all()]
         if user.id not in team_members:
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
```
Comment (redmyt) [inline eventually/vote/views.py]: maybe it is better to use this `access denied` response?

File: eventually/vote/views.py [hunk 3]
```diff
@@ -215,14 +219,14 @@ def get(self, request, team_id=None, event_id=None, vote_id=None,
...
```
Comment (redmyt) [inline eventually/vote/views.py]: maybe it is better to use this `access denied` response?

File: eventually/vote/views.py [hunk 4]
```diff
@@ -164,18 +168,18 @@ def delete(self, request, team_id, event_id, vote_id=None):  # pylint: disable=u
         team = Team.get_by_id(team_id)
 
         if not vote_id:
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
 
         members = [user.id for user in team.members.all()]
         if user.id not in members:
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
```
Comment (redmyt) [inline eventually/vote/views.py]: maybe it is better to use this `access denied` response?

File: eventually/vote/views.py [hunk 5]
```diff
@@ -115,18 +119,18 @@ def put(self, request, team_id, event_id, vote_id=None):  # pylint: disable=unus
...
 
         if not vote_id:
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
 
         members = [user.id for user in team.members.all()]
         if user.id not in members:
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
```
Comment (redmyt) [inline eventually/vote/views.py]: maybe it is better to use this `access denied` response?

File: eventually/vote/views.py [hunk 6]
```diff
@@ -46,14 +50,14 @@ def get(self, request, team_id=None, event_id=None, vote_id=None):  # pylint: di
...
```
Comment (redmyt) [inline eventually/vote/views.py]: maybe it is better to use this `access denied` response?

File: eventually/vote/views.py [hunk 7]
```diff
@@ -215,14 +220,14 @@ def get(self, request, team_id=None, event_id=None, vote_id=None,
 
         team_members = [user.id for user in team.members.all()]
         if user.id not in team_members:
-            return HttpResponse(status=400)
+            return RESPONSE_403_ACCESS_DENIED
 
         answer = Answer.get_by_id(answer_id)
         if not answer:
```
Comment (lhalam) [inline eventually/vote/views.py]:
```
if answer:
              answer = answer.to_dict()
               return JsonResponse(answer, status=200)
return RESPONSE_404_OBJECT_NOT_FOUND
```

**Review by lhalam — DISMISSED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**